### ISSUE TITLE
Add minuniquesize/maxuniquesize to the command --fastx_filter

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#225]
If a user wants to remove low-abundant sequences, it seems logical to try the command `--fastx_filter` first:
```sh
vsearch \
    --fastx_filter input.fasta \
    --sizein \
    --sizeout \
    --minuniquesize 2 \
    --fastaout filtered.fasta
```
As of now, abundance-based filtering is tied to more complex operations (e.g., dereplication, sorting).

**Comment by torognes** [comment]
I'll add this for the next release.

**Comment by torognes** [comment]
Added `minsize` and `maxsize` options to `fastx_filter`. Since `minsize` is shorter I haven chosen that name for the option. It is also used for sorting, while `minuniquesize` is used with dereplication.

**Comment by torognes** [comment]
Fixed in version 2.5.0.
